src/common/app.config.js

```diff
@@ -38,7 +38,7 @@ const appConfig = /* @ngInject */ function (envServiceProvider, $compileProvider
       development: {
         apiUrl: 'https://give-stage2.cru.org',
         imgDomain: '',
-        imgDomainDesignation: 'https://give-stage2.cru.org',
+        imgDomainDesignation: 'https://localhost.cru.org:9000',
```

Comment:
Did you mean to update this? This would require you to run a local AEM (which would actually be at `http://localhost:4502` by default)

Reply:
I did my testing on `https://localhost.cru.org:9000`, and since the code is now fetching the images, there are CORS errors if they're on a different origin. The localhost:9000 dev server proxies requests to `give-stage2`, which is why this works: https://github.com/CruGlobal/give-web/blob/2109c3c80116b6c7456e8a73b2c851630270f7c5/webpack.config.js#L143-L160